src/tools/miri/tests/pass/alloc-access-tracking.rs

```diff
@@ -0,0 +1,28 @@
+#![feature(start)]
+#![no_std]
+//@compile-flags: -Zmiri-track-alloc-id=17 -Zmiri-track-alloc-accesses -Cpanic=abort
```

Comment:
As an isolated no_std test, I think the alloc IDs here should be fully stable.

Reply:
Damn, somehow it is different on macOS... I guess I'll have to add some `only-target`.